account_chatter/models/account_journal.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2020 Vauxoo
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import models, fields
+
+
+class AccountJournal(models.Model):
+
+    _inherit = "account.journal"
```

Comment:
Why this? Does the journal include chatter by default in newer versions? AFAIK, no

Reply:
Yes, now in V13 the journal include [chatter](https://github.com/odoo/odoo/blob/13.0/addons/account/models/account.py#L681) by default